neighbors/src/helpers/apiCommunication.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { collection, onSnapshot, doc } from 'firebase/firestore';
+import { useEffect, useState, createContext } from 'react';
+import { PropTypes } from 'prop-types';
+
+import { db } from '../api/firebase';
+export const usersData = collection(db, 'Users');
+export const needHelpPostsData = collection(db, 'Posts-need-help');
+export const offerHelpPostsData = collection(db, 'Posts-offer-help');
+
+export const ThemeContext = createContext();
+export const ThemeProvider = ({ children }) => {
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    onSnapshot(doc(db, 'Users', 'zpiNSV5oKSGw9DVMTy4a'), (doc) => {
+      setUser({ id: 'zpiNSV5oKSGw9DVMTy4a', ...doc.data() });
```

Comment:
`id` powinno być w jakiś sposób wydzielone, choćby jako const przed komponentem, a najlepiej w osobnym w pliku z configiem

Reply:
Poprawione - Teraz jest to generowane poprzez pobieranie danych zautoryzowanego usera.